app/views/UtrPartnershipView.scala.html

```diff
@@ -48,12 +48,15 @@ <h1 class="govuk-heading-xl">@messages("utrPartnership.heading")</h1>
             ))
         </p>
 
+        <h2 class="govuk-heading-m">@messages("utrPartnership.heading.2")</h2>
```

Comment:
I would use something different here, e.g. utrParthership.h2.1, or only utrParthership.h2 since there is only one H2 element.